TheraLang.DAL/Entities/UserDetails.cs

```diff
@@ -17,7 +17,7 @@ public UserDetails()
         public string ShortInformation { get; set; }
         public string ImageURl { get; set; }
         public User User { get; set; }
-        public int Age { get; set; }
+        public DateTime BirthdayDate { get; set; }
```

Comment:
I think BirthDate is better name for this field

Reply:
okay

Reply:
guys,  I'll change it in user edit pr, okay? Because here I added thit property to the DTOs also and will rename everyvere in one pr